src/Core/Events/EventHandlers.cs

```diff
@@ -19,6 +19,7 @@ namespace Microsoft.Fast.Components.FluentUI;
 [EventHandler("onclosecolumnoptions", typeof(EventArgs), enableStopPropagation: true, enablePreventDefault: true)]
 [EventHandler("ontooltipdismiss", typeof(EventArgs), enableStopPropagation: true, enablePreventDefault: true)]
 [EventHandler("onsizechanged", typeof(EventArgs), enableStopPropagation: true, enablePreventDefault: true)]
+[EventHandler("onclickwithtargetid", typeof(TargetIdMouseEventArgs), enableStopPropagation: true, enablePreventDefault:true)]
```

Comment:
Following the `onclickwith...` naming scheme, I think using `MouseEventArgsWithTargetId` as a name makes more sense and is more clear that is a "MouseEventArgs+"